app/package.json

```diff
@@ -40,10 +37,7 @@
     "cypress": "^6.8.0",
     "fetch-mock": "^6.5.1",
     "fetch-readablestream": "^0.2.0",
-    "mocha": "^5.0.1",
-    "mocha-headless-chrome": "^2.0.3",
-    "sinon": "^7.3.2",
-    "webpack": "^4.1.1"
+    "sinon": "^7.3.2"
   },
   "resolutions": {
     "ssri": ">=8.0.1",
```

Comment:
also check if these resolutions need porting - if I remember correctly, some of them are there because of mocha